pkgs/top-level/all-packages.nix

```diff
@@ -26411,9 +26411,15 @@ with pkgs;
 
   gpsprune = callPackage ../applications/misc/gpsprune { };
 
-  gpu-screen-recorder = callPackage ../applications/video/gpu-screen-recorder { };
+  gpu-screen-recorder = callPackage ../applications/video/gpu-screen-recorder {
+    # rm me as soon as this package gains the support for cuda 11
+    cudatoolkit = cudatoolkit_10;
```

Comment:
I couldn't find means to remove `sm_30` from the gencode list, so building with cuda10 for now

CC @babbaj